api/utxoCache.ts

```diff
@@ -0,0 +1,109 @@
+import { getAddressUtxoOrdinalBundles, getUtxoOrdinalBundle } from './ordinals';
+
+export type UtxoStates = 'inscribed' | 'notInscribed' | 'unknown';
+
+export type UtxoCacheStruct = {
+  [utxoId: string]: UtxoStates;
```

Comment:
I wonder if it won't be better and more useful if we make this:
```suggestion
  [utxoId: string]: UtxoOrdinalBundle;
```

That bundle info would actually be a lot more useful and it's not a huge payload. It will also never change in the long run (unless we do a cache version change for something like sattributes), so it's safe to store long term for confirmed/indexed UTXOs.